src/robusta/core/discovery/utils.py

```diff
@@ -0,0 +1,59 @@
+from typing import List
+from kubernetes.client import V1Deployment, V1DaemonSet, V1StatefulSet, V1Job, V1Pod, V1ResourceRequirements
+
+from ...core.model.pods import PodResources, ResourceAttributes
+
+# This file contains utility related to k8s python api objects (rather than hikaru)
+
+
+def extract_containers_images(resource) -> List[str]:
+    """Extract images from k8s python api object (not hikaru)"""
+    try:
+        containers = []
+        if isinstance(resource, V1Deployment) \
+                or isinstance(resource, V1DaemonSet) \
+                or isinstance(resource, V1DaemonSet) \
+                or isinstance(resource, V1StatefulSet) \
+                or isinstance(resource, V1Job):
+            containers = resource.spec.template.spec.containers
+        elif isinstance(resource, V1Pod):
+            containers = resource.spec.containers
+
+        return [container.image for container in containers]
+    except Exception:
+        pass
+    return []
+
+
+def k8s_pod_requests(pod: V1Pod) -> PodResources:
+    """Extract requests from k8s python api pod (not hikaru)"""
+    return pod_resources(pod, ResourceAttributes.requests)
+
+
+def k8s_pod_limits(pod: V1Pod) -> PodResources:
```

Comment:
could not find a use for this function for some reason

Reply:
removing